wls-gui-wahllokalsystem/src/components/wlsComponents/WlsHeartbeat.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-menu>
+    <template #activator="{ props: menuActivator }">
+      <v-btn
+        v-bind="menuActivator"
+        :icon="getIcon(isOffline)"
+        variant="text"
+        density="comfortable"
+        size="x-large"
+        :color="getColor(isOffline)"
+      ></v-btn>
+    </template>
+    <v-card
+      width="250"
+      max-height="280"
+    >
+      <v-list class="pt-0">
+        <v-list-item>
+          <v-btn
+            @click.stop="heartbeat"
+            :disabled="isCheckStatusBtnDisabled"
+            color="primary"
+            style="width: 100%"
```

Comment:
Der Event-Handler sollte am nach den anderen Properties kommen

```suggestion
            :disabled="isCheckStatusBtnDisabled"
            color="primary"
            style="width: 100%"
            @click.stop="heartbeat"
```